<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBTreePage</class>
 <widget class="QWidget" name="RBTreePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1844</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>红黑树实例</string>
  </property>
  <widget class="QLabel" name="RBzjspng">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1600</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black; background-color:white</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Graph绘制区域&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="RBinsert">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>80</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RBinputInsertPair">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RBerase">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>160</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RBinputEraseKey">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="RBinput_shunxucharu">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RBshunxucharu">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>240</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>顺序插入</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RBinput_suijicharu">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>280</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RBsuijicharu">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>320</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>随机插入</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RBinput_zidongsuijicharu">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>360</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RBzidongsuijicharu">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>400</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>自动随机插入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RBfanhuizhuye">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>750</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主页</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
